src/components/calcite-action-menu/calcite-action-menu.tsx

```diff
@@ -142,6 +143,17 @@ export class CalciteActionMenu {
     this.updateActions(this.actionElements);
   }
 
+  // --------------------------------------------------------------------------
+  //
+  //  Methods
+  //
+  // --------------------------------------------------------------------------
+
+  @Method()
+  async setFocus(): Promise<void> {
+    focusElement(this.open ? this.actionElements[0] || this.menuEl : this.menuButtonEl);
```

Comment:
Would there be cases where we want a `focusId` for the menu button?

Reply:
I don't think so. It depends on if it's open or not so we can go with that for now.